homeassistant/components/wled/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
```

Comment:
The location of the WLED repo has changed btw, so we should update the release URL